util/stmtsummary/variables_test.go

```diff
@@ -14,108 +14,108 @@
 package stmtsummary
 
 import (
-	. "github.com/pingcap/check"
+	"testing"
+
 	"github.com/pingcap/tidb/config"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = Suite(&testVariablesSuite{})
-
-type testVariablesSuite struct {
-}
-
-func (s *testVariablesSuite) TestSetInVariable(c *C) {
+func TestSetInVariable(t *testing.T) {
+	t.Parallel()
 	sv := newSysVars()
 	st := sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(config.GetGlobalConfig().StmtSummary.MaxStmtCount))
+	require.Equal(t, int64(config.GetGlobalConfig().StmtSummary.MaxStmtCount), st)
 
 	err := sv.setVariable(typeMaxStmtCount, "10", false)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	st = sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(10))
+	require.Equal(t, int64(10), st)
 	err = sv.setVariable(typeMaxStmtCount, "100", false)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	st = sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(100))
+	require.Equal(t, int64(100), st)
 	err = sv.setVariable(typeMaxStmtCount, "10", true)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	st = sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(10))
+	require.Equal(t, int64(10), st)
 	err = sv.setVariable(typeMaxStmtCount, "100", true)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	st = sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(100))
+	require.Equal(t, int64(100), st)
 	err = sv.setVariable(typeMaxStmtCount, "10", false)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	st = sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(100))
+	require.Equal(t, int64(100), st)
 	err = sv.setVariable(typeMaxStmtCount, "", true)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	st = sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(10))
+	require.Equal(t, int64(10), st)
 	err = sv.setVariable(typeMaxStmtCount, "", false)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	st = sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(config.GetGlobalConfig().StmtSummary.MaxStmtCount))
+	require.Equal(t, int64(config.GetGlobalConfig().StmtSummary.MaxStmtCount), st)
 }
 
-func (s *testVariablesSuite) TestSetBoolVariable(c *C) {
+func TestSetBoolVariable(t *testing.T) {
+	t.Parallel()
 	sv := newSysVars()
 	en := sv.getVariable(typeEnable)
-	c.Assert(en > 0, Equals, config.GetGlobalConfig().StmtSummary.Enable)
+	require.Equal(t, config.GetGlobalConfig().StmtSummary.Enable, en > 0)
 
 	err := sv.setVariable(typeEnable, "OFF", false)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	en = sv.getVariable(typeEnable)
-	c.Assert(en > 0, Equals, false)
+	require.Equal(t, false, en > 0)
```

Comment:
`require.LessOrEqual(t, en, 0)` is better?

Reply:
hi， all this kind is fixed, thanks.

Reply:
hi @mmyj , unit test failed due to Elements  type not match, one is int64 and the other is int.
i am search testify api to see if there are some method can do the compare but  *ignore* the int type, otherwise seems a convert is required. 
if  so, they orginal one seems more simple.
what your idea?

Reply:
I think convert the `int` to `int64` can resolve this issue.

Reply:
ok, fixed, please check